SQLZOO/DDLGeneralElection.sql

```diff
@@ -0,0 +1,25 @@
+--DDL General Election
+
+DROP TABLE IF EXISTS ge;
+CREATE TABLE ge(
```

Comment:
I don't know if it made you name this table `ge` but that name isn't very illuminating.